packages/num/num.1.0/opam

```diff
@@ -39,3 +39,14 @@ url {
   src: "https://github.com/ocaml/num/archive/v1.0.tar.gz"
   checksum: "md5=a4c359108d0b885bf43909e705b21291"
 }
+post-messages: [
+  "Installation failure with the message 'The installation of num failed at \"make findlib-install\"'
+is due to a known packaging bug with the num package. As a workaround run the following two commands,
+then try to reinstall the target package again:
+
+  ocamlfind remove num
+  ocamlfind remove num-top
```

Comment:
If this is a common issue why not run these before each install instead?

Reply:
I'm not sure how frequent the problem is. It only hits me about once every couple of months. I'm happy with that tho, if it seems sensible! I'll wait until the additional reviewer you've tagged in weighs in :)

Thanks for the suggestion!